### PR TITLE
setters methods on ol.style.*

### PR DESCRIPTION
**Pull request by quentin-ol**
Please fill in this template.

- [ ] Make your PR against the `master` branch.
- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [ ] Run `npm run lint -- package-name` if a `tslint.json` is present.

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Run `tsc` without errors.
- [ ] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

**Review by osechet — CHANGES_REQUESTED** [review]
Some of your modifications reflect changes done in newer versions of openlayers, v3.19.0 for some and v3.20.0 for others). You must update the version of the definition file accordingly. In this case v3.20.0.

File: openlayers/index.d.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Type definitions for OpenLayers v3.20.0
-// Project: http://openlayers.org/
+// Project: http:/openlayers.org/
```
Comment (andy-ms) [inline openlayers/index.d.ts]: `http://` is standard, no need to change.

**Comment by quentin-ol** [comment]
My bad, didn't mean to change RFC 7230.

File: openlayers/index.d.ts (hunk 1 above)
Comment (quentin-ol) [inline openlayers/index.d.ts]: My bad, didn't mean to change RFC 7230.

**Pull request merged**